go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kyma-project/cloud-manager
 
-go 1.22.5
+go 1.22.7
```

Comment:
Why is Go upgraded?

Reply:
change go.mod `google.golang.org/grpc v1.65.0 -> google.golang.org/grpc v1.68.0`
run `go mod tidy`